libbeat/cfgfile/cfgfile.go

```diff
@@ -120,11 +128,17 @@ func Load(path string) (*common.Config, error) {
 		return nil, err
 	}
 
-	return common.MergeConfigs(
+	config, err = common.MergeConfigs(
 		defaults,
 		config,
 		overwrites,
 	)
+	if err != nil {
+		return nil, err
+	}
+
+	debugf("return config =>\n%v", config.DebugString())
```

Comment:
We should change this to something like: "Complete merged configuration:" as `return config` is not really explaining on what it is.